src/app/test_executive/test_executive.ml

```diff
@@ -401,8 +403,12 @@ let main inputs =
         let%bind () = Malleable_error.List.iter non_seed_pods ~f:start_print in
         [%log info] "Daemons started" ;
         [%log trace] "executing test" ;
-        T.run network dsl )
+        let%bind result = T.run network dsl in
+        let open Malleable_error.Let_syntax in
+        let%bind () = Engine.Network_manager.destroy net_manager network in
```

Comment:
Destroy was already called by cleanup. Why does it need to also be called here now?

Reply:
Yes, looks like it need to be run in such way. If i remove this code then no coverage action is being done

Reply:
You've put the code coverage code in the wrong place.